api/apps/api/src/modules/access-control/projects-acl/dto/update-user-role-project.dto.ts

```diff
@@ -0,0 +1,14 @@
+import { Roles } from '@marxan-api/modules/users/role.api.entity';
```

Comment:
It might be a good moment to move this entity directly to `access-control` module and get rid of its dependencies in other modules. Reasoning: we have dedicated module now, so there isn't any reason left to keep polluting other modules (like projects, users) with anything about roles or access control.